modules/querying/pages/data-types.adoc

```diff
@@ -596,16 +596,23 @@ When the query where the `FILE` was declared finishes running, the `FILE` conten
 ==== Define s3 file object
 The path should start with `s3://`, followed by the bucket name and the S3 path, e.g., `s3://bucket-name/queryoutput/output.csv`. During the execution of the query, content will be uploaded to the S3 bucket. Note that the S3 object cannot be modified or appended, if an S3 object with the same path already exists, it will be overwritten.
 
-==== Set S3 connection credentials
-To use an S3 path, ensure that the necessary permissions and configurations are properly set up to allow read/write access to the specified S3 bucket. Configure the S3 connection credentials using `gadmin config`:
-[source,bash]
+==== Using Session Parameters
+Use the following session parameters in GSQL:
```

Comment:
```suggestion
The S3 credentials can be set as GSQL session parameters, so they they persist for a user for a full session.
```
The description above hints at the meaning of "session parameter" in case you or the reader do not understand.